engine/src/flutter/lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -454,109 +450,6 @@ class CanvasKitRenderer implements Renderer {
   ui.ParagraphBuilder createParagraphBuilder(ui.ParagraphStyle style) =>
       isExperimentalWebParagraph ? WebParagraphBuilder(style) : CkParagraphBuilder(style);
 
-  // TODO(harryterkelsen): Merge this logic with the async logic in
-  // [EngineScene], https://github.com/flutter/flutter/issues/142072.
```

Comment:
Should we close this issue?

Reply:
I will include it in the list of issues closed by this PR